docs/tutorials/AddRemark.adoc

```diff
@@ -0,0 +1,425 @@
+= Tutorial - Adding a new Command
```

Comment:
I think this file shouldn't be here?

Reply:
Hm I'm not sure why it's showing up as an addition... it was merged from our team repo :/